pages/upload/index.tsx

```diff
@@ -110,7 +110,7 @@ export default function Upload() {
             value={youtubeUrl}
             onChange={onChangeYoutubeUrl}
           />
-          <YouTubePlayer videoId={makeYouTubeVideoId(youtubeUrl)} isAutoPlay={true} />
+          <YouTubePlayer videoId={makeYouTubeVideoId(youtubeUrl) || ''} isAutoPlay={true} />
```

Comment:
이건 취향 문제라서 꼭 반영은 안해도 되는데, props에 이렇게 직접 defaultValue를 넣어주는 것 보다는 YouTubePlayer 컴포넌트 내부에서 props 받아오는 쪽에 직접 해당 값에 대한 defaultValue를 넣어주는게 더 깔끔하지 않을까요? 예를 들면 이런 느낌..

```suggestion
          <YouTubePlayer videoId={makeYouTubeVideoId(youtubeUrl)} isAutoPlay={true} />
```

```ts
// YouTubePlayerButton.tsx

// ...

interface YouTubePlayerProps {
  videoId?: string
  isAutoPlay: boolean
  isInMemoryList?: boolean
}
const YouTubePlayer: React.FC<YouTubePlayerProps> = ({ videoId = '', isAutoPlay, isInMemoryList }) => {

// more code ...

```